asciidoctorj-documentation/src/main/asciidoc/integrator-guide.adoc

```diff
@@ -988,30 +957,24 @@ include::src/test/java/org/asciidoctor/integrationguide/extension/CopyrightFoote
 ----
 <1> All Preprocessors must extend the class `org.asciidoctor.extension.Postprocessor` and implement the method `process()`.
```

Comment:
Not related to your PR, but could you fix this? 😁 
All ~Preprocessors~ Postprocessors must extend the class `org.asciidoctor.extension.Postprocessor`

Reply:
I can tell from this one, that you did go through all the text in defail